File: src/main/java/ni/edu/uam/michimaker_api/dto/UsuarioDto.java
package ni.edu.uam.michimaker_api.dto;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class UsuarioDto {

    private Integer id;

    private String username;

    private String nombre;

    // Dentro de tu clase UsuarioDto.java añade:
    private String biografia;

    private String correo;

    private String password;

    private String fotoPerfil;

    public UsuarioDto() {
    }
}

File: app/src/main/java/ni/edu/uam/michimaker/dto/UsuarioDto.kt
// UsuarioDto.kt
package ni.edu.uam.michimaker.dto

data class UsuarioDto(
    val id: Int? = null,
    val username: String,
    val nombre: String,
    val biografia: String? = null, // <- Nuevo campo
    val correo: String,
    val password: String? = null,
    val fotoPerfil: String? = null
)